integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,15 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	testRequires(c, NativeExecDriver)
```

Comment:
Brian, 
>Why does this require native? It looks like you have this in the LXC template.
could you let me know what is expected from me here.. will change accordingly..